data.yml

```diff
@@ -1843,32 +1855,93 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/lendingRateAccountMetadata"
+        termDeposit:
+          type: array
+          required: 
+            - lodgementDate
+            - maturityDate
+            - maturityInstructions
+          properties: 
+            lodgementDate:
+              type: string
+              description: The lodgement date of the original deposit. 
+            maturityDate:
+              type: string
+              description: Maturity date for the term deposit. 
+            maturityAmount: 
+              type: string
+              description: Amount to be paid upon maturity. If absent it implies the amount to paid is variable and cannot currently be calculated. 
```

Comment:
Add a comma: "If absent, it implies the amount..."

Reply:
Fixed.